database/user.go

```diff
@@ -222,7 +271,7 @@ func (db *DB) UserSetStripeId(ctx context.Context, u *User, stripeId string) err
 
 // UserSetTier sets the user's tier to the given value.
 func (db *DB) UserSetTier(ctx context.Context, u *User, t int) error {
-	if t <= TierMinReserved || t >= TierMaxReserved {
+	if t <= TierAnonymous || t >= TierMaxReserved {
```

Comment:
Should this have changed?

Reply:
The initial idea was to have `TierMinReserved` be the low guard value but then we needed to introduce a level for anonymous users, so I repurposed the value. The alternative was updating all values in the code and in the database but I didn't see a need to do that. We can do it in the future in case we need to use the low guard value for some reason.